locales/base/translation.json

```diff
@@ -1923,5 +1931,9 @@
       "title": "You need more {{feeTokenSymbol}} for gas",
       "description": "Add {{feeTokenSymbol}} to cover your gas fees or adjust the amount you are sending"
     }
+  },
+  "networkName": {
+    "ethereum": "Ethereum",
+    "celo": "Celo"
```

Comment:
We don't already have this anywhere else?
cc @cajubelt @satish-ravi

Reply:
I couldn't find it in the translations, i suppose we can use a hard coded value if we prefer that. idk if blockchain names can be translated (i imagined that they could?)

Reply:
ah, i found what you mean https://github.com/valora-inc/wallet/blob/main/src/shared/conts.ts#L3 - i updated to use this because it's already there but i'm not sure that this is better than having it in the transaction file (i can imagine for languages like Chinese, there would be a translation for these network names).